example/migrations/0002_auto_20160513_0857.py

```diff
@@ -9,7 +9,6 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('contenttypes', '0002_remove_content_type_name'),
```

Comment:
I had removed it because I thought the dependency was breaking the older django versions (<1.7) on test. I think I'm wrong though.